roles/common/apache2/apache2_install/templates/apache2.conf.j2

```diff
@@ -0,0 +1,45 @@
+# Include module configuration:
+Include /etc/apache2/mods-enabled/*.load
+Include /etc/apache2/mods-enabled/*.conf
+
+# Include ports listing
+Include /etc/apache2/ports.conf
+
+{% if apache2_servername is defined %}
+ServerName {{apache2_servername}}
```

Comment:
For the sake of formatting consistency, there needs to be spaces around the variable inside the brackets